packages/docker-build/src/commands/build.ts

```diff
@@ -116,8 +111,8 @@ export default class DockerBuildCommand extends BaseCommand {
         });
 
         await xfs.mktempPromise(async (cwd) => {
-          const manifestDir = ppath.join(cwd, toFilename('manifests'));
-          const packDir = ppath.join(cwd, toFilename('packs'));
+          const manifestDir = ppath.join(cwd, 'manifests');
+          const packDir = ppath.join(cwd, 'packs');
```

Comment:
`toFilename` was removed but not mentioned in the changelog. The recommendation is to cast to `Filename` instead where needed.